### ISSUE TITLE
Error adding setting XMLParser

### ISSUE THREAD

**Issue by RazuvaevDD** [RazuvaevDD/ProjectWork2019Neiro#29]
**Entry conditions**
_Settings.xml_
```
<root>
    <setting id="1">
        <str>STRING</str>
        <dbl>228.600000</dbl>
        <b>0</b>
        <i>1488</i>
    </setting>
    <setting id="3">
        <str>STRING</str>
        <dbl>228.600000</dbl>
        <b>0</b>
        <i>1488</i>
    </setting>
</root>
```
_main.cpp_
```
Settings_module::XMLParser p("Settings.xml");
p.loadXML();
p.addSetting(s);
```
**expected result**
_Settings.xml_
```
<root>
    <setting id="1">
        <str>STRING</str>
        <dbl>228.600000</dbl>
        <b>0</b>
        <i>1488</i>
    </setting>
    <setting id="3">
        <str>STRING</str>
        <dbl>228.600000</dbl>
        <b>0</b>
        <i>1488</i>
    </setting>
    <setting id="4">
        <str>STRING</str>
        <dbl>228.600000</dbl>
        <b>0</b>
        <i>1488</i>
    </setting>
</root>
```
or
```
<root>
    <setting id="1">
        <str>STRING</str>
        <dbl>228.600000</dbl>
        <b>0</b>
        <i>1488</i>
    </setting>
    <setting id="3">
        <str>STRING</str>
        <dbl>228.600000</dbl>
        <b>0</b>
        <i>1488</i>
    </setting>
    <setting id="2">
        <str>STRING</str>
        <dbl>228.600000</dbl>
        <b>0</b>
        <i>1488</i>
    </setting>
</root>
```

**current result**
_Settings.xml_
```
<root>
    <setting id="1">
        <str>STRING</str>
        <dbl>228.600000</dbl>
        <b>0</b>
        <i>1488</i>
    </setting>
    <setting id="3">
        <str>STRING</str>
        <dbl>228.600000</dbl>
        <b>0</b>
        <i>1488</i>
    </setting>
    <setting id="3">
        <str>STRING</str>
        <dbl>228.600000</dbl>
        <b>0</b>
        <i>1488</i>
    </setting>
</root>
```

**Comment by AverinLV** [comment]
@RazuvaevDD 
Fixed in a919e59

**Comment by RazuvaevDD** [comment]
Ok
